reproschema/models/model.py

```diff
@@ -1,16 +1,15 @@
 from __future__ import annotations
-from datetime import datetime, date
-from decimal import Decimal
+
+from datetime import datetime
 from enum import Enum
-import re
-import sys
-from typing import Any, List, Literal, Dict, Optional, Union
+from typing import Dict, List, Optional, Union
+
 from pydantic.version import VERSION as PYDANTIC_VERSION
 
 if int(PYDANTIC_VERSION[0]) >= 2:
-    from pydantic import BaseModel, ConfigDict, Field, field_validator
+    from pydantic import BaseModel, ConfigDict, Field
 else:
-    from pydantic import BaseModel, Field, validator
+    from pydantic import BaseModel, Field
```

Comment:
I do not think this is needed as we are pinned to pydantic >= 2

Reply:
@djarecka

Reply:
it is possible, this part was automatically generated by the linkml

Reply:
OK let's leave for now then.

Reply:
but for now I would leave it like this